src/main/java/org/opensearch/knn/index/codec/KNN990Codec/NativeEngines990KnnVectorsWriter.java

```diff
@@ -78,8 +86,12 @@ public void flush(int maxDoc, final Sorter.DocMap sortMap) throws IOException {
                 field.getVectors()
             );
 
+            // TODO: Extract quantization state here, uncomment below line once implemented
+            // quantizationStateWriter.writeState(field.getFieldInfo().getName(), quantizationState);
```

Comment:
Should remove the comments now?

Reply:
I need to wait until after Vikash's next PR is merged to uncomment.  The quantizationState variable doesn't exist yet

Reply:
make sense.